mutate/mutate_fuzzer.go

```diff
@@ -0,0 +1,179 @@
+// +build gofuzz
+/*
+ * umoci: Umoci Modifies Open Containers' Images
+ * Copyright (C) 2021 SUSE LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mutate
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	fuzz "github.com/AdaLogics/go-fuzz-headers"
+	"github.com/opencontainers/go-digest"
+	imeta "github.com/opencontainers/image-spec/specs-go"
+	ispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/opencontainers/umoci/oci/cas"
+	casdir "github.com/opencontainers/umoci/oci/cas/dir"
+	"github.com/opencontainers/umoci/oci/casext"
+)
+
+// setup() does the necessary setup for the fuzzer
+// It takes a data parameter provided by the fuzzer
+func setup(dir string, data []byte) (cas.Engine, ispec.Descriptor) {
```

Comment:
You can't call this function `setup` -- it breaks the integration tests build because there is a function called `setup` in `mutate/mutate_test.go`.